Functions/CatalogItems/Out-RsFolderContent.ps1

```diff
@@ -74,7 +74,7 @@ function Out-RsFolderContent
     {
         if(($item.TypeName -eq 'Folder') -and $Recurse)
         {
-            $relativePath = $item.Path.Replace($Path, "")
+            $relativePath = $item.Path.Replace($Path, "\")
```

Comment:
I was double checking the code and I think the problem is not the character that is replaced but that  string.replace is case sensitive and that is affecting the replacement, can you try with this one (is not case sensitive)
$relativePath = $item.Path -replace $Path, ""

Also when debugging you can enable verbose that helps to see the messages ( I should update the readme with it)
$VerbosePreference = "continue"

Assume you have a folder called BI Pro (which have inside BI Pro/subfolder) 
Out-RsFolderContent -ReportServerUri http://myserver/reportserver -Path "/bi pro" -Destination c:\temp\bipro

$items = Get-RsCatalogItems -proxy:$Proxy -Path:"/bi pro"
($items | where TypeName -eq "Folder")[0].Path
Will return the following folder path (see the capital from the server)
/BI Pro/SubFolder
And then the code try to remove the /BI Pro part so the relative path is only /SubFolder , but the current implementation fails to do because the case sensitivity.

can you give it a try to the recommended changes and update the PR